tests/baselines/reference/deeplyNestedAssignabilityIssue.errors.txt

```diff
@@ -65,7 +65,6 @@
         }
     }
     
-Found 2 errors in 1 file.
+Found 2 errors in 1 file: tests/cases/compiler/deeplyNestedAssignabilityIssue.ts:22

```

Comment:
Should this have the line number?

Reply:
Personally, I prefer the line number because it will take you to the first error - I'm totally open to removing it if you think others will interpret it as a bug?

Reply:
Yeah, that is actually compelling, but I definitely assumed it was a bug. Maybe

```
Found {0} errors in the same file, starting at {1}
```

Reply:
I have greyed out the `:23` (everywhere) now, and made that change 👍🏻